### PR TITLE
add new subpage connectYourAccount

### PR DESCRIPTION
**Pull request by Ervige**

### CODE REVIEW

File: pages/housePrideQuiz/connectYourAccount.html [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+        <p class="article__paragraph">
+            Like any Hogwarts student, you’ll need a wand and a Hogwarts house to progress your school life. But if you’re registered with us, you can actually 
+            <a href="#" class="article__link">bring your wizarding credentials into the game</a>
+            without having to get sorted or pick a wand again.
+        </p>
+        <h1 class="article__headline">My Hogwarts house is sacred! How do I link accounts right now?</h1>
+        <p class="article__paragraph">
+            Never fear. Here’s a handy step-by-step.
+        </p>
+        <o class="article__list">
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/connectYourAccount.html:100]: I am sorry, but I do not know <o> tag. Please fix it.

File: pages/housePrideQuiz/connectYourAccount.html [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+              />
+            </figure>
+            <hr class="writtenBy__line" />
+          </div>
+        </div>
+      </div>
+    </main>
+    <footer>
+      <div class="mobileFooter">
+        <div class="mobileFooter__logo">
```
Comment (AlpacaInTheCode) [inline pages/housePrideQuiz/connectYourAccount.html:194]: Wrap  in <figure> tag

**Review by AlpacaInTheCode — CHANGES_REQUESTED** [review]
Fix all issues

File: pages/housePrideQuiz/connectYourAccount.html (hunk 1 above)
Comment (Ervige) [reply pages/housePrideQuiz/connectYourAccount.html:100]: fixed

File: pages/housePrideQuiz/connectYourAccount.html (hunk 2 above)
Comment (Ervige) [reply pages/housePrideQuiz/connectYourAccount.html:194]: fixed

**Review by AlpacaInTheCode — APPROVED** [review]

**Pull request merged**